x-pack/test/functional/apps/ml/anomaly_detection/annotations.ts

```diff
@@ -7,96 +7,336 @@
+      analysis_config: {
+        bucket_span: '15m',
+        influencers: [],
+        detectors: [
+          {
+            function: 'mean',
+            field_name: 'responsetime',
+          },
+        ],
+      },
+      data_description: { time_field: '@timestamp' },
+      analysis_limits: { model_memory_limit: '10mb' },
+      model_plot_config: { enabled: true },
+    };
+    // @ts-expect-error doesn't implement the full interface
+    const DATAFEED_CONFIG: Datafeed = {
+      datafeed_id: `datafeed-${jobId}`,
+      indices: ['ft_farequote'],
+      job_id: jobId,
+      query: { bool: { must: [{ match_all: {} }] } },
+    };
+    const annotation = {
+      timestamp: 1455142431586,
+      end_timestamp: 1455200689604,
+      annotation: 'Test annotation',
+      job_id: jobId,
+      type: 'annotation' as Annotation['type'],
+      detector_index: 0,
+      event: 'user',
+      create_time: 1626129498464,
+      create_username: 'user1',
+      modified_time: 1626129498464,
+      modified_username: 'user2',
+    };
+
     before(async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
       await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
       await ml.testResources.setKibanaTimeZoneToUTC();
 
       await ml.api.createAndRunAnomalyDetectionLookbackJob(JOB_CONFIG, DATAFEED_CONFIG);
+
       // Points the read/write aliases of annotations to an index with wrong mappings
       // so we can simulate errors when requesting annotations.
-      await ml.testResources.setupBrokenAnnotationsIndexState(JOB_CONFIG.job_id);
       await ml.securityUI.loginAsMlPowerUser();
     });
 
     after(async () => {
       await ml.api.cleanMlIndices();
     });
 
-    it('displays error on broken annotation index and recovers after fix', async () => {
-      await ml.testExecution.logTestStep('loads from job list row link');
-      await ml.navigation.navigateToMl();
-      await ml.navigation.navigateToJobManagement();
+    describe('creating', function () {
+      const newText = 'Created annotation text';
+
+      it('creates annotation', async () => {
+        await ml.testExecution.logTestStep('loads from job list row link');
+        await ml.navigation.navigateToMl();
+        await ml.navigation.navigateToJobManagement();
+
+        await ml.jobTable.waitForJobsToLoad();
+        await ml.jobTable.filterWithSearchString(JOB_CONFIG.job_id, 1);
+
+        await ml.jobTable.clickOpenJobInSingleMetricViewerButton(JOB_CONFIG.job_id);
+        await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+
+        await ml.testExecution.logTestStep('pre-fills the job selection');
+        await ml.jobSelection.assertJobSelection([JOB_CONFIG.job_id]);
+
+        await ml.testExecution.logTestStep('pre-fills the detector input');
+        await ml.singleMetricViewer.assertDetectorInputExist();
+        await ml.singleMetricViewer.assertDetectorInputValue('0');
```

Comment:
I think we don't need these validations here as they're not specific to this test. We already check these things as part of our single metric viewer tests, so we don't need to repeat them in other tests that have a different focus.

Reply:
Updated here https://github.com/elastic/kibana/pull/105527/commits/ec07c2c1e48391ce4acbdba4ddc5d8d8fa4593f5